code/game/objects/items/weapons/RPD.dm

```diff
@@ -518,50 +520,62 @@ var/global/list/RPD_recipes=list(
 		show_menu(usr)
 
 
-/obj/item/weapon/pipe_dispenser/afterattack(atom/A, mob/user)
```

Comment:
... isn't ideal how? it takes turfs as arguments just fine, same as afterattack.